gradle/formatting.gradle

```diff
@@ -81,8 +81,7 @@ allprojects {
               '',
               '\\#java|\\#org.opensearch|\\#org.hamcrest|\\#'
           )
-
-          eclipse().withP2Mirrors(Map.of("https://download.eclipse.org/", "https://mirror.umd.edu/eclipse/")).configFile rootProject.file('buildSrc/formatterConfig.xml')
+          eclipse().withP2Mirrors(['https://download.eclipse.org/eclipse/updates/4.29/':'https://ftp.osuosl.org/pub/eclipse/eclipse/updates/4.29/']).configFile rootProject.file('buildSrc/formatterConfig.xml')
```

Comment:
This can be?
```suggestion
          eclipse().withP2Mirrors(Map.of("https://download.eclipse.org/eclipse/updates/4.29/", "https://ftp.osuosl.org/pub/eclipse/eclipse/updates/4.29/")).configFile rootProject.file('buildSrc/formatterConfig.xml')
```

Reply:
Initially we use `Map.of` which works, but just following the instructions in https://github.com/diffplug/spotless/blob/main/plugin-gradle/README.md#eclipse-jdt

Reply:
@prudhvigodithi Map worked too https://github.com/opensearch-project/neural-search/pull/1540/files